examples/franka_blockstacking_example.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+
+import numpy as np
+from display_utils import display_rgb
+
+import habitat
+from habitat.core.logging import logger
+
+SENSOR_KEY = "rgb"
+CONFIG_FILE = "configs/tasks/franka_point.yaml"
+BLOCK_CONFIGS_PATH = "data/test_assets/objects/nested_box"
```

Comment:
This glb is mainly intended for testing. You should use the "cubeSolid" primitive instead.

From the interactivity tutorial:
```
# Do this once:
# Get a default cube primitive template
cube_handle = obj_attr_mgr.get_template_handles("cubeSolid")[0]
cube_template_cpy = obj_attr_mgr.get_template_by_handle(cube_handle)
# Modify the template's configured scale
cube_template_cpy.scale = np.array([0.2, 0.2, 0.2])
# Register the modified template under a new name
obj_attr_mgr.register_template(cube_template_cpy, "my_scaled_cube")

#Do this when you want a new instance:
occluder_obj = rigid_obj_mgr.add_object_by_template_handle("my_scaled_cube")
```